Berksfile

```diff
@@ -1,3 +1,4 @@
 source "https://supermarket.chef.io"
 
 metadata
+cookbook 'socrata_configuration', git: 'git@git.socrata.com:socrata_configuration-cookbook.git'
```

Comment:
On the one hand, "open source by default" and all but, on the other, seems like there's a some stuff in here that's highly specific to Socrata, making the cookbook not of much use to anyone external.

Is the Clortho/KMS bit something that could be split out into an private cookbook, or are the actual resources being called something that could be open sourced?